README.md

```diff
@@ -521,6 +521,16 @@ Setting `--v=4` or greater on the Descheduler will log all reasons why any pod i
 Pods subject to a Pod Disruption Budget(PDB) are not evicted if descheduling violates its PDB. The pods
 are evicted by using the eviction subresource to handle PDB.
 
+## Metrics
```

Comment:
Can you update the table of contents too? (we should add a verify script for that...)
Other than that, lgtm
/approve

Reply:
Done